README.md

```diff
@@ -0,0 +1,82 @@
+# Chief
+
+A simple command pattern for Ruby.
+
+[![Code Climate](https://codeclimate.com/github/policygenius/chief/badges/gpa.svg)](https://codeclimate.com/github/policygenius/chief)
+
+
+## Usage
+
+A `Chief::Command`'s primary interface is the `.call` method, executes the command and returns a `Chief::Result` instance.
+
+A `Chief::Result` encapsulates whether the command succeeding, including the return values and (optionally) any errors encountered.
+
+Suppose we have an ExampleApplication, that needs to notify an external service with a message. First, we create the command:
+
+```Ruby
+require 'net/http'
+
+module ExampleApplication
+  module SomeExternalService
+    class Notify < Chief::Command
+      attr_reader :message
+
+      def initialize(message)
+        @message = message
+      end
+
+      def call
+        response = report_to_external_service
+
+        if response.code == 200
+          success!(message)
+        else
+          fail!(false, 'Could not notify the external service')
```

Comment:
Those are possible (see https://github.com/policygenius/chief/blob/master/spec/chief/command_spec.rb#L55). I'd welcome a PR that updates the README!